### DISCUSSION TITLE
How to uninstall completely?

### DISCUSSION THREAD

**Discussion by robertburke14** [kyleneideck/BackgroundMusic#672] [Q&A] [answered]
Hi, I uninstalled the app by dragging it to trash but Background Music still appears as an option in the sound menu and under output & input in System Settings > Sound. I also see Background Music and Background Music (UI Sounds) listed under System Settings > Sound Effects > Play sound effects through.

How can I remove these entries?

**Comment by kyleneideck** [comment, marked as answer]
https://github.com/kyleneideck/BackgroundMusic#uninstall
